spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/mustache/MustacheProperties.java

```diff
@@ -42,6 +43,26 @@
 	 */
 	private String suffix = DEFAULT_SUFFIX;
 
+	/**
+	 * Default value for any variable that is missing.
+	 */
+	private String defaultValue;
+
+	/**
+	 * Value for any variable that resolves to null.
+	 */
+	private String nullValue;
+
+	/**
+	 * Handles converting objects to strings.
+	 */
+	private final Formatter formatter = new Formatter();
```

Comment:
This needs to be an implementation of the `Mustache.Formatter` interface. Unless we require that implementation to have a default constructor and treat the property as the fully-qualified name of a class that we instantiate, I don't think it can be configured using properties.